erpnext/accounts/doctype/period_closing_voucher/period_closing_voucher.json

```diff
@@ -6,39 +6,32 @@
  "engine": "InnoDB",
  "field_order": [
   "transaction_date",
-  "posting_date",
+  "company",
   "fiscal_year",
-  "year_start_date",
+  "period_start_date",
+  "period_end_date",
   "amended_from",
-  "company",
   "column_break1",
   "closing_account_head",
-  "remarks",
   "gle_processing_status",
+  "remarks",
   "error_message"
  ],
  "fields": [
   {
+   "default": "Today",
    "fieldname": "transaction_date",
    "fieldtype": "Date",
    "label": "Transaction Date",
    "oldfieldname": "transaction_date",
    "oldfieldtype": "Date"
   },
-  {
-   "fieldname": "posting_date",
-   "fieldtype": "Date",
-   "label": "Posting Date",
-   "oldfieldname": "posting_date",
-   "oldfieldtype": "Date",
-   "reqd": 1
-  },
```

Comment:
Removing a mandatory field from a stable major version, hidden in a `perf` PR is unexpected, breaking behavior.